src/wasm-lib/kcl-test-server/src/lib.rs

```diff
@@ -69,9 +69,19 @@ fn start_worker(i: u8, engine_addr: Option<String>) -> mpsc::Sender<WorkerReq> {
     // Make a work queue for this worker.
     let (tx, mut rx) = mpsc::channel(1);
     tokio::task::spawn(async move {
-        let state = ExecutorContext::new_for_unit_test(UnitLength::Mm, engine_addr)
-            .await
-            .unwrap();
+        let state = ExecutorContext::new_with_client(
```

Comment:
I think this should still be `new_for_unit_test` right, that method didn't get deleted, it's modified below to use the new stuff.

Reply:
new with client is different and i am going to use it in kcl.py

Reply:
new with unit test defaults some of the shit

Reply:
oh i see never mind ill fix